### DISCUSSION TITLE
TOUCH DIDNT WORK ON ILI9341, ESP32 S3 WROOM1 IS USED

### DISCUSSION THREAD

**Discussion by diephauthan** [Bodmer/TFT_eSPI#3412] [Q&A - Touch]
"I had an ESP32 S3 WROOM 1 and an ILI9341 display working perfectly, but I could not use the touch on the screen. I used the touch_calibrate example, when I ran the example I didn't touch the arrow but it automatically moved to the next corner, and the screen displayed "touch screen to test!", I tried to touch the screen but nothing happened. This is my definition in user_setup."

```
#define TFT_CS   10 //     10 or 34 (FSPI CS0) 
#define TFT_MOSI 11 //     11 or 35 (FSPI D
#define TFT_SCLK 12 //     12 or 36 (FSPI CLK)
#define TFT_MISO 13 //     13 or 37 (FSPI Q)

// Use pins in range 0-31
#define TFT_DC    7
#define TFT_RST   6

#define TOUCH_CS 5 // Optional for touch screen
```

**Comment by diephauthan** [comment]
As you can see on Logic 2, the orange zone is for touch corners as indicated, and the blue zone is when the screen displayed 'touch screen to test!' Even though I didn't touch the screen, the CS_Touch was still active.
![image](https://github.com/user-attachments/assets/38e512bf-dc43-4042-a240-e34d55c84757)

**Comment by TangerineDreamer** [comment]
I suggest you leave MISO on the air,and erase the little FS calibrate file before doing a new caibration,or why not,change
the name of the file in the beginning og the sketch.
This problem often occurs with the MISO pin ,maybe it doesn't go tristate on some displays,so try with MISO -> NOTHING
